kolibri/plugins/coach/assets/src/views/plan/LessonEditDetailsPage/EditDetailsResourceListTable.vue

```diff
@@ -164,7 +164,9 @@
         this.$emit('update:resources', resources);
       },
       resourceChannelTitle(id) {
-        const match = this.$store.getters['getChannelObject'](this.contentNodeMap[id].channel_id);
+        const match = this.$store.getters['getChannelObject'](
+          (this.contentNodeMap[id] || {}).channel_id
+        );
```

Comment:
This looks like a probable bug:

```
(this.contentNodeMap[id] || {}).channel_id
```

because it will sometimes evaluate to `undefined`.

If that is in fact intended behavior, there might be a more clear way to handle this.